### PR TITLE
Fix incorrect avatar background colour when using a custom theme

### PR DESCRIPTION
**Pull request by jdauphant**
"--avatar-background-colors_*" are not taken in account in some situation, by using "getComputedStyle" we ensure it's taken in account in all case.

## Problem Details

When you add variables at the body level like that for a custom theme :

https://github.com/tchapgouv/tchap-web-v4/blob/6f62e1536754468b7c03ff7542dcdfb352de512d/res/themes/tchap-light/css/_tchap_custom_vars.pcss#L36-L40
image

We can see the variable here :
image

But as the style isn't computed, document.body.style.getPropertyValue("--avatar-background-colors_1"); don't return the value.
getComputedStyle(document.body) do the computation of the style and return the good answer :
image

## Checklist

-   [x] Tests written for new code (and old code if feasible)
-   [x] Linter and other CI checks pass
-   [x] Sign-off given on the changes (see [CONTRIBUTING.md](https://github.com/matrix-org/matrix-react-sdk/blob/develop/CONTRIBUTING.md))

Signed-off-by: Julien DAUPHANT [julien.dauphant@beta.gouv.fr](mailto:julien.dauphant@beta.gouv.fr)

---
Here's what your changelog entry will look like:

## 🐛 Bug Fixes
 * Fix incorrect avatar background colour when using a custom theme ([\#10598](https://github.com/matrix-org/matrix-react-sdk/pull/10598)). Contributed by @jdauphant.

### CODE REVIEW

**Comment by jdauphant** [comment]
I am waiting for the CI approval

**Comment by richvdh** [comment]
> I am waiting for the CI approval

looks like it's been approved now.

Please could you update the PR summary and description to be a bit clearer about the symptoms of the bug that is being fixed? Currently it explains *what* you are changing, but doesn't explain *why*.

**Comment by jdauphant** [comment]
@richvdh I have include more details, is it good for you ?

**Review by richvdh — APPROVED** [review]
looks good to me. Thank you for contributing back!

**Pull request merged**
